src/commonTest/kotlin/baaahs/show/LayoutValidatorSpec.kt

```diff
@@ -0,0 +1,63 @@
+package baaahs.show
+
+import baaahs.describe
+import baaahs.gl.override
+import baaahs.toBeSpecified
+import ch.tutteli.atrium.api.fluent.en_GB.containsExactly
+import ch.tutteli.atrium.api.fluent.en_GB.isEmpty
+import ch.tutteli.atrium.api.verbs.expect
+import org.spekframework.spek2.Spek
+
+object LayoutValidatorSpec : Spek({
+    describe<LayoutValidator> {
+        val layoutValidator by value { LayoutValidator() }
```

Comment:
This `by value` thing creates a new instance for every spec — for each `it` block.